net-im/toxic/toxic-0.8.3.ebuild

```diff
@@ -0,0 +1,84 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python3_4 python3_5 python3_6 )
+
+inherit python-single-r1 xdg
+
+DESCRIPTION="A curses-based client for Tox"
+HOMEPAGE="https://github.com/JFreegman/toxic"
+SRC_URI="https://github.com/JFreegman/toxic/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+X +sound notification +python +qrcode +video"
+REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} )"
+
+# Not a typo; net-libs/tox only has a 'both or neither' option
+RDEPEND="
+	net-libs/tox:0/0.2
```

Comment:
You'll most likely have to update this ebuild, so you can sort these unconditional deps, too.

Reply:
Also, this is not how subslots are usually meant to be used. Most likely this should really be `>=net-libs/tox-0.2.5:=`